### PR TITLE
[automated] Rename MaterialComponentsAlpha to MaterialComponentsBeta.

### PR DESCRIPTION
**Pull request by jverkoey**
This is an automated change generated by replacing all instances of MaterialComponentsAlpha with MaterialComponentsBeta. This is not a breaking change because changes to Alpha/Beta components (including renaming them) are not considered breaking.

The MaterialComponentsAlpha podspec was mistakenly named Alpha, when what we meant was more close to Beta. The distinction is that Alpha components are not expected to be used by clients, while Beta components are.

### CODE REVIEW

File: contributing/beta_components.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
+# Beta program for components
```
Comment (romoore) [inline contributing/beta_components.md:1]: Was this a `git mv` operation that git couldn't track? If it was a copy-paste, can you perform it as a move so we preserve the file history?

**Review by romoore — APPROVED** [review]
LGTM with just one comment regarding the renamed markdown file.

File: contributing/beta_components.md (hunk 1 above)
Comment (jverkoey) [reply contributing/beta_components.md:1]: Looks like git didn't track the rename. Will fix.

Comment (jverkoey) [reply contributing/beta_components.md:1]: Git doesn't like the fact that we're changing so much of the file unfortunately.

Comment (romoore) [reply contributing/beta_components.md:1]: Silly git...

**Pull request merged**